ui/packages/consul-ui/app/controllers/dc/services/show/topology.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { getOwner } from '@ember/application';
+
+export default class TopologyController extends Controller {
+  get routeInstance() {
```

Comment:
* this shouldn't be a getter if it's not accessing any dynamic class member like tracked properties or other getters or args
* since we are not using it in many spots it would make sense to co locate with the invoke it like:

```js
@action
createIntention(source, destination) {
const route = getOwner(this).lookup('route:dc.services.show.topology');
    return route.createIntention(source, destination);
  }
```